api/CanvasRenderingContext2D.json

```
@@ -20,7 +20,7 @@
             "version_added": "1.5"
           },
           "firefox_android": {
-            "version_added": null
+            "version_added": true
```

Probably safe to set this to the initial version as well

```suggestion
            "version_added": "4"
```